crypto/modes/siv128.c

```diff
@@ -0,0 +1,372 @@
+/*
+ * Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Copyright 2017, Akamai Technologies. All Rights Reserved.
```

Comment:
Remove the akamai copyright; put Daniel's name in the commit message.

Reply:
I included a reference to his github; but the code has diverged considerably.